executor/runtime/types/pod.go

```diff
@@ -549,6 +557,9 @@ func (c *PodContainer) Runtime() string {
 	return DefaultOciRuntime
 }
 
+// SeccompAgentEnabledForNetSyscalls only represents whether a user set the SeccompAgentEnabledForNetSyscalls
```

Comment:
If so, why would we leave this here and not always use network mode?

Reply:
I gotta wait till we are not using the attribute everywhere?

I still need *some* function in here to tell me if the attribute is set.
Once spinnaker and stuff can set network mode, I'll depreciate usage of this attribute asap and remove these things.